### DISCUSSION TITLE
cannot boot from USB after installing Surface kernel

### DISCUSSION THREAD

**Discussion by unclemu85** [linux-surface/linux-surface#506] [Q&A]
Now that I've installed Linux kernel on my Surface 3, I discovered that device will no longer boot from USB. I am trying to change the distribution from Elementery to Lubuntu.

**Comment by qzed** [comment]
That sounds like it'd be a UEFI issue. Is there any way to select the boot device manually in the UEFI menu? If not you could try using `efibootmgr` to switch the boot order and place USB first.

**Reply by unclemu85** [reply]
Thanks for the response. Here's what worked. I tried several bootable USBs, and the one that worked was the one by Elementary OS. After a fresh install, the other bootable USBs seemed to work. 

I must've broke something with the original Elementary OS as I was trying to load the correct kernel for Surface 3.
